test/envs/regional-dr.yaml

```diff
@@ -32,6 +32,7 @@ templates:
           - name: rook-cluster
           - name: rook-toolbox
           - name: rook-pool
+          - name: rook-cephfs
```

Comment:
How much time this takes?

Reply:
OK, found in the e2e job:

```
2024-05-09 01:47:02,857 INFO    [rdr-dr1/0] addons/rook-cephfs/start completed in 39.78 seconds
2024-05-09 01:47:06,327 INFO    [rdr-dr1/0] addons/rook-cephfs/test completed in 3.47 seconds
```

Looks good. I wonder if we should remove the csi-hostpath-driver and volumesnapshot addons which slow down the run by minutes (part of time to start the cluster):

2024-05-09 01:39:21,265 INFO    [rdr-hub] Cluster started in 70.08 seconds
2024-05-09 01:40:25,387 INFO    [rdr-dr2] Cluster started in 134.20 seconds
2024-05-09 01:41:21,252 INFO    [rdr-dr1] Cluster started in 190.06 seconds

Without these addons the clusters all cluster can start in ~90 seconds.

We can work on this later.

Reply:
On memory usage only inspection and time will tell, I did not face any issues in my usual runs (about 10-12 till now)